shim.h

```diff
@@ -170,5 +171,7 @@ extern X509 *X_sk_X509_value(STACK_OF(X509)* sk, int i);
 extern long X_X509_get_version(const X509 *x);
 extern int X_X509_set_version(X509 *x, long version);
 
+extern const char *get_default_cert_dir(void);
```

Comment:
maybe lets keep X_X509_ prefix?

Reply:
lol, so I dropped the `X509` prefix here, but it is present when calling the function. I guess that means the `shim` is not needed ? o_O